pkg/license/license.go

```diff
@@ -45,19 +47,29 @@ type LicensingResolver struct {
 
 // ToInfo returns licensing information given the total memory of all Elastic managed components
 func (r LicensingResolver) ToInfo(totalMemory resource.Quantity) (LicensingInfo, error) {
-	eru := inEnterpriseResourceUnits(totalMemory)
+	ERUs := inEnterpriseResourceUnits(totalMemory)
 	memoryInGB := inGB(totalMemory)
-	licenseLevel, err := r.getOperatorLicenseLevel()
+	operatorLicense, err := r.getOperatorLicense()
 	if err != nil {
 		return LicensingInfo{}, err
 	}
 
-	return LicensingInfo{
+	licenseLevel := r.getOperatorLicenseLevel(operatorLicense)
+	maxERUs := r.getMaxEnterpriseResourceUnits(operatorLicense)
+
+	licensingInfo := LicensingInfo{
 		Timestamp:               time.Now().Format(time.RFC3339),
 		EckLicenseLevel:         licenseLevel,
 		TotalManagedMemory:      memoryInGB,
-		EnterpriseResourceUnits: eru,
-	}, nil
+		EnterpriseResourceUnits: ERUs,
+	}
+
+	// include the max ERUs only for a non trial license
+	if !operatorLicense.IsTrial() {
```

Comment:
Operator license can be nil

Reply:
or `maxERUs > 0`

Reply:
I opted to check the nullity to keep the `IsTrial()` call.